examples/redux-saga-typescript-graphql-fluentui/README.md

```diff
@@ -0,0 +1,40 @@
+# redux-saga-typescript-graphQL-fluentUI example
+
```

Comment:
Can you please add more details about the example here under the title 🙏 , what it does, what libraries is it using, why is it using them and anything else that may be important to know about the example.

Reply:
yes, i will add more details